src/main/java/com/laborete/LaboreteAPI/profile/services/UsersServiceImpl.java

```diff
@@ -36,33 +43,53 @@ public class UsersServiceImpl implements UsersService {
     private static final String FILE_IS_EMPTY = "The file cannot be empty";
     private static final String FILE_NAME_IS_EMPTY = "The file name cannot be empty";
     private static final String FILE_FAILED_UPLOAD = "Failed to upload file: ";
+
```

Comment:
I would place all the constants together - no need for empty lines between them.